android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -424,33 +425,34 @@ protected void onSafeCreate(@Nullable Bundle savedInstanceState)
       addTask(new Factory.RestoreRouteTask());
   }
 
+  private void refreshLightStatusBar()
+  {
+    UiUtils.setLightStatusBar(this, !(
+        RoutingController.get().isPlanning()
+        || Framework.nativeIsInChoosePositionMode()
+        || ThemeUtils.isNightTheme(this)
```

Comment:
nit: for performance reasons, it's better to use the more frequent comparison first in OR clauses.